app/models/user.rb

```diff
@@ -94,6 +96,32 @@ def opt_into_newsletter_on_sign_up
     subscribe_to_mailchimp(true)
   end   
 
+  # this function checks if this user has completed Blank Slate training
+  def finished_training?
+    uri = URI.parse("https://app.blankslatetechnologies.com/sreo/v0.1/app/covidvolunteers/user/check-status")
+    request = Net::HTTP::Post.new(uri)
+    request.basic_auth("helpwithcovid@gmail.com", "C6XfTc12")
```

Comment:
it is kind of a pain, but we should probably be using env vars to do auths. with this, we have leaked the password for the helpwithcovid blank slate account. also realizing now this is not actually our email. I will send a message to Matt on slack to fix this.